scripts/licenses/licenses/git_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package licenses
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	git "gopkg.in/src-d/go-git.v4"
+)
+
+func TestGitFileURL(t *testing.T) {
+	dir, err := ioutil.TempDir("", "git_test")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(dir)
+
+	cloneOpts := git.CloneOptions{
+		URL:   "https://github.com/google/trillian",
+		Depth: 1,
+	}
+	if _, err := git.PlainClone(dir, false, &cloneOpts); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
If it's safe to assume that this test is being run from a Git checkout, then couldn't you just use `../../../LICENSE` as your test input and skip all the tempdir/cloning stuff?

Reply:
There's no knowing what remote URL is configured for "origin" in the local checkout. While this would work for a repo cloned using `go get`, it wouldn't work for a repo cloned from a fork, for example.